flight/PiOS/STM32F4xx/pios_internal_adc.c

```diff
@@ -296,7 +295,8 @@ static int32_t PIOS_INTERNAL_ADC_PinGet(uintptr_t internal_adc_id, uint32_t pin)
 	}
 
 	/* return accumulated result and clear accumulator */
-	result = accumulator[pin].accumulator / (accumulator[pin].count ?: 1);
+	result = accumulator[pin].accumulator /
+		(accumulator[pin].count ? accumulator[pin].count : 1);
```

Comment:
IMO it was more readable before on one line. :stuck_out_tongue:

Reply:
I broke it into a new line because the short-circuit ternary is a gnu extension that I'd rather we not use.

Reply:
Fair enough, agree we should avoid extensions where reasonable.